services/core/entities/demoReviewDayCounterEntity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity } from "dynamodb-toolbox";
+import { getTable } from "./getTable";
+
+export const getDemoReviewDayCounterEntity = (tableName: string): Entity => {
+  const myTable = getTable(tableName);
+
+  const demoReviewDayCounterEntity = new Entity({
+    name: "demoReviewDayCounterEntity",
+    attributes: {
+      PK: { partitionKey: true, hidden: true, prefix: "DATE#" },
+      SK: { sortKey: true, hidden: true, default: "DEMO_COUNTER" },
+      date: [
+        "PK",
+        0,
+        {
+          type: "string",
+          default: () => new Date().toDateString(),
+        },
+      ],
+      count: { type: "number" },
+    },
+    table: myTable,
+  } as const);
```

Comment:
I have a suggestion which I think will work better for the longer term. 
PK: DATE#... SK: DEMO, 

Add attributes: dailyCount: string.

I think the name for this should be ReviewDemoCounterEntity.

In future we can add rateLimited: string (boolean) which will determine if we are currently ratelimiting depending on the amount of requests.

Then for normal we would have. 
PK: DATE#...
SK: REPO#...
dailyCount: 

name is ReviewRepoCounterEntity